composer.json

```diff
@@ -25,11 +25,12 @@
         "psr/log": "^1.1",
         "yiisoft/aliases": "^3.0@dev",
         "yiisoft/arrays": "^3.0@dev",
+        "yiisoft/data": "^3.0@dev",
```

Comment:
I'd avoid this dependency. See the comment where it's used.

Reply:
It needs for 
https://github.com/yiisoft/view/pull/58/commits/1043fa5004b2d251519e74d1e143fe4d92f05411#diff-a3592cd657d9165af5eacaef68d15024R24

Reply:
LinkPager should be moved to https://github.com/yiisoft/yii-dataview

Reply:
And LinkSorter too?

Reply:
Yes.